strr-examiner-web/app/components/Host/Expansion/FilingHistory.vue

```diff
@@ -23,7 +23,9 @@ const { data: filingHistory, status } = await useLazyAsyncData<FilingHistoryEven
     } else {
       // for Registrations include Application and Registration histories
       const [applicationHistory, registrationHistory] = await Promise.all([
-        getApplicationFilingHistory((activeRecord.value as HousApplicationResponse).header.applicationNumber),
+        getApplicationFilingHistory(
+          (activeRecord.value as HousRegistrationResponse).header.applications[0].applicationNumber
```

Comment:
Can you elaborate a bit on this change?

Reply:
The RegistrationSerializer on the backend was updated, the response structure was changed as there can be multiple applications associated with a registration. For now, the examiner UI uses the most recent application associated with registration for showing details. This can change when the design is updated.